performance_test/helper_scripts/regression_checkers/ApexComparison.py

```diff
@@ -0,0 +1,554 @@
+        self.logger.info('REFERENCE FILE:   {}'.format(self.ref_file))
+        self.logger.info(
+            'Min Latency (ms): {:.5f}'.format(self.ref_latency_min)
+        )
+        self.logger.info('Max RSS (KB):     {}'.format(self.ref_max_rss))
+        self.logger.info(
+            'Max Jitter (ms):  {:.5f}'.format(self.max_ref_jitter)
+        )
+        self.logger.info('---------------------------------------------------')
+        self.logger.info('TARGET FILE:      {}'.format(self.target_file))
+        self.logger.info(
+            'Min Latency (ms): {:.5f}'.format(self.target_latency_min)
+        )
+        self.logger.info('Max RSS (KB):     {}'.format(self.target_max_rss))
+        self.logger.info(
+            'Max Jitter (ms):  {:.5f}'.format(self.max_target_jitter)
+        )
+        self.logger.info('---------------------------------------------------')
+        self.logger.info('COMPARISON:')
+        self.logger.info('{} Latency (ms): {:.5f} ({:.5f} %)'.format(
+            self.lat_analysis, self.lat_diff, self.lat_perc)
+        )
+        self.logger.info('{} RSS (KB):     {:.5f} ({:.5f} %)'.format(
+            self.rss_analysis, self.rss_diff, self.rss_perc)
+        )
+        self.logger.info('{} Jitter (ms):  {:.5f} ({:.5f} %)'.format(
+            self.ref_jitternalysis, self.jitter_diff, self.jitter_perc)
+        )
+        self.logger.info('---------------------------------------------------')
+        self.logger.info('FINAL RESULT')
+        if self.return_value == 0:
+            self.logger.info('Comparison passed =)')
+        else:
+            self.logger.warning('Comparison failed =(')
+        self.logger.info('---------------------------------------------------')
+        self.logger.debug('')
+
+    def _print_start(self):
+        """Print and log the configuration."""
+        self.logger.debug('')
+        self.logger.info('###################################################')
+        self.logger.info('       RUNNING COMPARISON WITH CONFIGURATION       ')
+        self.logger.info('###################################################')
+        self.logger.info('Reference file: {}'.format(self.ref_file))
+        self.logger.info('Target file: {}'.format(self.target_file))
+        self.logger.info(
+            'Columns of interest: {}'.format(self.columns_of_interest)
+        )
+        self.logger.info(
+            'Latency threshold: {}'.format(self.latency_threshold)
+        )
+        self.logger.info('RSS threshold: {}'.format(self.rss_threshold))
+        self.logger.info('Jitter threshold: {}'.format(self.jitter_threshold))
+        self.logger.info('Print results: {}'.format(self.print_enable))
+        self.logger.info('###################################################')
+
+    def compare(self):
+        """
+        Compare the files.
+
+        :return: The result of the analysis.
+            0 if comparison yields better results than the reference
```

Comment:
If the only values are 0 or 1, just use a boolean here.

Reply:
Fixed